operators/multiclusterobservability/pkg/config/config.go

```diff
@@ -482,6 +482,18 @@ func GetDefaultTenantName() string {
 
 // GetObsAPIHost is used to get the URL for observartium api gateway.
 func GetObsAPIHost(client client.Client, namespace string) (string, error) {
+	mco := &observabilityv1beta2.MultiClusterObservability{}
```

Comment:
nit, but I wonder if the `GetObsAPIHost` shouldn't be adapted to accept a context here itself?

Reply:
I'll add this in a follow up PR where I will do the same as here but for the alertmanager URL, then both functions will get the context. 👍